src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -719,7 +739,8 @@ def _update_issue(self, new_comment=None, notify=True):
                                       issueId=str(self.id)))
     return result
 
-  def save(self, new_comment=None, notify=True):
+  @retry_on_invalid_gaia_accounts
+  def save(self, skip_emails=[], new_comment=None, notify=True):
```

Comment:
nit: add `skip_emails` to the very end, in case there are callers that are relying on argument order.